### ISSUE TITLE
Writing Good Commits: missing blank lines

### ISSUE THREAD

**Issue by Mariatta** [coala/coala#3248]
_From @jayvdb on October 30, 2016 7:11_

```
$ rstcheck ./Developers/Writing_Good_Commits.rst
./Developers/Writing_Good_Commits.rst:117: (INFO/1) Possible title underline, too short for the title.
```

The above error is misleading.  The problem is that an extra blank line is needed.

_Copied from original issue: coala/documentation#248_

**Comment by aptrishu** [comment]
Are you working on this issue? @Cygnus32

**Comment by HyungKen** [comment]
Could you assign this issue to me?

**Comment by jayvdb** [comment]
Fixed in ee7e30ca
